app/models/ont/plate_with_samples_factory.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+# Ont namespace
+module Ont
+  # PlateWithSamplesFactory
+  # A factory for bulk inserting a plate and all of its dependents
+
+  # rubocop:disable Metrics/ClassLength
+  class PlateWithSamplesFactory
+    include ActiveModel::Model
+
+    validate :check_plate_factory
+
+    def initialize(attributes = {})
+      @attributes = attributes
+      time = DateTime.now
+      @timestamps = { created_at: time, updated_at: time }
+    end
+
+    def process
+      @plate_factory = Ont::PlateFactory.new(attributes)
+    end
+
+    def save(**options)
+      return false unless options[:validate] == false || valid?
+
+      @serialised_plate_data = plate_factory.bulk_insert_serialise(self, validate: false)
```

Comment:
More an observation than something that absolutely needs updating, but the naming of this method is probably less relevant now it's not obvious this class is the bulk inserter.  Could probably simplify it to just `serialize`.